linux/kernel/building.md

```diff
@@ -125,6 +125,11 @@ If you are using a 32-bit operating system (for example, our Raspberry Pi Deskto
 
 `sudo apt install zlib1g-dev:amd64`
 
+If you are using CCache and a CI environment, instruct it to not use compiler's mtime for cache ID calculations.
+This is because Git intentionally doesn't save file timestamps, so each time you clone the toolchain, its file mtimes are different, invalidating CCache's cache with default settings.
+
+`ccache --set-config=compiler_check=content`
```

Comment:
Maybe this should go _between_ the previous two lines? :man_shrugging:

Reply:
I was afraid that this would make it unclear which text relates to which code -- since the previous sections are always "chunk of text - chunk of code". If there's no such problem, fine by me.